doc/classes/RichTextLabel.xml

```diff
@@ -628,6 +628,9 @@
 		<member name="language" type="String" setter="set_language" getter="get_language" default="&quot;&quot;">
 			Language code used for line-breaking and text shaping algorithms, if left empty current locale is used instead.
 		</member>
+		<member name="mb_click_scrolling_rate" type="float" setter="set_mb_click_scrolling_rate" getter="get_mb_click_scrolling_rate" default="0.5">
+			Middle mouse button click-and-drag scrolling rate, in text lines (using default normal font and font size) per pixel per second.
```

Comment:
In text lines... per pixel per second? It's a bit long-winded and I'm not sure what it means at a glance.